Common/include/option_structure.hpp

```diff
@@ -1334,10 +1334,37 @@ enum FLAMELET_SCALAR_SOURCES {
  */
 enum FLAMELET_LOOKUP_OPS {
   TD,       /*!< \brief Thermochemical properties (temperature, density, diffusivity, etc.). */
+  PD,       /*!< \brief Preferential diffusion scalars. */
```

Comment:
TD and PD are not very descriptive abbreviations. Maybe better to use:
PD -> PREFDIF
TD ->THERMO